test/e2e/broker_defaults_webhook_test.go

```diff
@@ -129,12 +130,12 @@ func TestBrokerNamespaceDefaulting(t *testing.T) {
 			Resource(schema.GroupVersionResource{Group: "eventing.knative.dev", Version: "v1", Resource: "brokers"}).
 			Namespace(c.Namespace).
 			Create(ctx, obj, metav1.CreateOptions{})
-		assert.Nil(t, err)
+		require.Nil(t, err)
```

Comment:
What do you think about updating all `assert.Nil` in other e2e test?

Reply:
Yes, good point. If this usage makes sense then it would be worth it to audit the other e2e tests as well. As you mentioned an easy one to start with is the channel_defaults_webhook_test.go

Reply:
Created an issue for the other relevant test #4739